release.nix

```diff
@@ -11,76 +11,71 @@
 let
   config = {
     packageOverrides = pkgs:
-      let
-        python_protobuf3_0 =
-          (pkgs.pythonPackages.protobufBuild pkgs.protobuf3_0).override {
-            doCheck = false;
-          };
-      in
-        { haskellPackages = pkgs.haskellPackages.override {
-            overrides = haskellPackagesNew: haskellPackagesOld: rec {
+      { haskellPackages = pkgs.haskellPackages.override {
+          overrides = haskellPackagesNew: haskellPackagesOld: rec {
 
-              aeson =
-                pkgs.haskell.lib.dontCheck (haskellPackagesNew.callPackage ./nix/aeson.nix { });
+            aeson =
```

Comment:
A lot of these Haskell package overrides are probably no longer necessary now that you've upgraded to a more recent Nixpkgs version.  Most of them were just to get a more recent version that we needed back when we were using an older default Haskell package set